config.properties

```diff
@@ -0,0 +1,21 @@
+#      Copyright (c) Microsoft Corporation.
```

Comment:
@git4rk this technique of using maven properties and filtering to substitute values is what we use with WebLogic as well.